### PR TITLE
Prepare repository for December 2020 release

### PR DESCRIPTION
**Pull request by dominiklohmann**
###  :notebook_with_decorative_cover: Description

This must be rebased before merging. The merge commit of this branch is the one I'll tag for release.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

n/t

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,10 +33,9 @@ Every entry has a category for which we use the following visual abbreviations:
   delay before the command completed.
   [#1234](https://github.com/tenzir/vast/pull/1234)
 
-- 🎁 A new tracepoint `meta_index_lookup(us, candidates)` can be used, that
-  is called whenever a meta index lookup is completed. It gets two arguments,
-  the number of microseconds spent resolving the lookup and the number of
-  candidates returned.
+- 🎁 A new tracepoint `meta_index_lookup(us, candidates)` exists for tracing
+  completed meta index lookups. Its two arguments are the number of microseconds
+  spent resolving the lookup and the number of candidate partitions returned.
```
Comment (mavam) [inline CHANGELOG.md:38]: Can we remove this entirely? The changelog is for users, not developers.

File: Dockerfile_prebuilt [hunk 2]
```diff
@@ -30,6 +30,9 @@ RUN chown -R vast:vast /var/log/vast /var/lib/vast
 WORKDIR /var/lib/vast
 VOLUME ["/var/lib/vast"]
 
+RUN ls /opt/vast
+RUN ls $PREFIX
+
```
Comment (mavam) [inline Dockerfile_prebuilt:35]: Should this be really in here?

**Review by mavam — APPROVED** [review]
Checked the diff in detail; LGTM

**Review by tobim — APPROVED** [review]
Git grep doesn't reveal any other occurrences of the previous version. Good to go.

**Pull request merged**